pkg/clients/ec2/securitygroup.go

```diff
@@ -152,6 +153,9 @@ func GenerateIPPermissions(objectPerms []ec2.IpPermission) []v1beta1.IPPermissio
 		}
 		permissions[i] = ipPerm
 	}
+	sort.Slice(permissions, func(i, j int) bool {
```

Comment:
Nit: this works find but perhaps consider a new func that sorts permissions and returns it. Eg:

155      }
159      return sortBtPort(*permissions)
160 }